app/controllers/InterstitialController.scala

```diff
@@ -153,9 +153,11 @@ class InterstitialController @Inject() (
     }
   }
 
-  def displayNewsAndUpdates: Action[AnyContent] = authenticate { implicit request =>
+  def displayNewsAndUpdates(lang: String): Action[AnyContent] = authenticate { implicit request =>
```

Comment:
I don't think you need lang here. If you extend i18n in the class definition you should be able to extract the current lang.

Or even better do this in the newsAndTilesConfig. See here: https://github.com/hmrc/taxcalc-frontend/blob/d079c1ffee5236df02497ba792cab26ce8032f6b/app/config/ApplicationConfig.scala#L44

Reply:
Will try this.